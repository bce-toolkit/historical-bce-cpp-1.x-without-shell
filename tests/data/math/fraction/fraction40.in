4973368147199224483095098631233554927080
1401177035616097129021871502059362818187
33234147590584756264683985761630257695
2297178950790651972369817678272401256771
