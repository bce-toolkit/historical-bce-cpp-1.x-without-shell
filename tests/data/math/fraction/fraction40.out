11471283546680966679410179998539254955199395733335074741691622018000917503957645/3218754392548541992793638772300481775723514853242557525452892456107607175694177
11378149697876559868179788000789717932594964393788020201581192294804385018559715/3218754392548541992793638772300481775723514853242557525452892456107607175694177
55095217008777359928459500496161564786926630542985172664591009547625777960200/1072918130849513997597879590766827258574504951080852508484297485369202391898059
2284943324455752654758996799932897288779436012712309494327281431280530252251736/9313384880440681123039199774953702260443133954705454011042972319653248539793
0
1
0
0
1
1
